src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/TimeSeriesViewer.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import { getDefaultStoryDecorator } from '../../../../Models/Services/StoryUtilities';
+import TimeSeriesViewer from './TimeSeriesViewer';
+import { ITimeSeriesViewerProps } from './TimeSeriesViewer.types';
+import MockAdapter from '../../../../Adapters/MockAdapter';
+import { DataHistoryExplorerContext } from '../../DataHistoryExplorer';
+
+const wrapperStyle = { width: '600px', height: '400px' };
+
+export default {
+    title: 'Components/TimeSeriesViewer',
+    component: TimeSeriesViewer,
+    decorators: [getDefaultStoryDecorator<ITimeSeriesViewerProps>(wrapperStyle)]
+};
+
+type TimeSeriesViewerStory = ComponentStory<typeof TimeSeriesViewer>;
+
+const Template: TimeSeriesViewerStory = (args) => {
+    return (
+        <DataHistoryExplorerContext.Provider
+            value={{ adapter: new MockAdapter() }}
+        >
+            <TimeSeriesViewer {...args} />{' '}
+        </DataHistoryExplorerContext.Provider>
+    );
+};
+
+export const Mock = Template.bind({}) as TimeSeriesViewerStory;
```

Comment:
can we show the no data state too? and if there's an error state, maybe that'd make a nice story

Reply:
There is an "Empty" story already, no error states yet.